tutorials/install-cockroachdb-scaleway-instances/index.mdx

```diff
@@ -0,0 +1,480 @@
+| Completely closed    | Does not create a console backend and frontend on the Load Balancer. In this case, a machine with SSH access to a node could use an SSH tunnel to access the DB Console.|
+
+## Creating Instances
+
+This procedure shows you how to create Instances for each node you plan to have in your cluster. Make sure to replace all example values with your own.
+
+<Message type="important">
+  To [ensure survivability](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#topology), you must run at least three nodes.<br /><br /> You can use any Instance type except for PLAY2-PICO Instances as they have 1 vCPU, which is below CockroachDB's minimum requirement of 4 vCPUs.<br /><br /> For more details, refer to [the Cockroach hardware recommendations](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#hardware) and [cluster topology](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#topology).
+</Message>
+
+1. Run the following command to create a Private Network:
+    ```
+    scw vpc private-network create name=cockroachdb region=fr-par subnets.0=192.168.0.0/24
+    ```
+
+2. Create a Public Gateway to access Instances.
+    ```
+    scw vpc-gw gateway create name=external-access type=VPC-GW-S enable-bastion=true
+    ```
+
+3. Create a DHCP subnet that matches the one used in your Private Network.
+    ```
+    scw vpc-gw dhcp create subnet=192.168.0.0/24
+    ```
+
+4. Configure your gateway with your Private Network using your newly created DHCP subnet.
+    ```
+    scw vpc-gw gateway-network create enable-dhcp=true enable-masquerade=true \
+    gateway-id=<id> dhcp-id=<id> private-network-id=<id>
+    ```
+
+5. Run the following command to create an Instance in the `fr-par-1` zone with a data volume of 30G.
+    ```
+    scw instance server create name=db01 zone=fr-par-1 type=PRO2-XXS image=ubuntu_jammy ip=none additional-volumes.0=b:30G
+    ```
+
+6. Create a private NIC (Network Interface Card) for your Instance.
+    ```
+    scw instance private-nic create zone=fr-par-1 server-id=<id> private-network-id=<id>
+    ```
+
+7. Repeat steps 5 and 6 to create the two other nodes in the `fr-par-2` and `fr-par-3` AZs.
+
+4. Check your Instance's DHCP entry on the Private Network and retrieve its private IP.
+    ```
+    scw vpc-gw dhcp-entry list gateway-network-id=<id>
+    ```
+
+5. Connect to the Instances using the gateway bastion.
+    ```
+    ssh -J bastion@<gw-ip>:61000 root@db01.priv
+    ```
+
+## Synchronizing clocks
+
+CockroachDB requires moderate levels of [clock synchronization](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#clock-synchronization) to preserve data consistency.
+
+For this reason, nodes spontaneously shut down when they detect that their clock is out of sync with at least half of the other nodes in the cluster by 80% of the maximum offset allowed (500ms by default).
```

Comment:
```suggestion
For this reason, nodes spontaneously shut down when they detect that their clock is out of sync with at least half of the other nodes in the cluster by 80% of the maximum offset allowed (500ms by default). This is done to avoid consistency anomalies. However, it is best to prevent clocks from drifting too far off. To do so, a clock synchronization software can be run on each mode.
```